src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -357,17 +440,6 @@ public class VmrUpdater : VmrManagerBase, IVmrUpdater
                 _logger.LogWarning(e.Message);
                 continue;
             }
-            catch (Exception)
```

Comment:
Why we no longer want to have this generic exception catch?

Reply:
We catch that one level above